app/controllers/facility_journals_controller.rb

```diff
@@ -30,12 +30,17 @@ def new_with_search
     set_default_variables
     @layout = "two_column_head"
 
-    respond_to do |format|
-      format.csv do
-        # used for "Export as CSV" link for order details with expired accounts
-        @order_details = @order_details.expired_account
+    @after_search_hook = lambda do
+      @valid_order_details, @invalid_order_details = ValidatorFactory.partition_valid_order_details(@order_details.unexpired_account)
+      @invalid_order_details += @order_details.expired_account
```

Comment:
Is there anything we need to do around sorting these together since it'll be two sets appended to each other?